corehq/apps/sms/migrations/0051_reset_modified_on.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.24 on 2021-06-29 13:16
+
+from django.db import migrations
+from django.db.models import Min
+
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+def _reset_date_modified(model_name):
+    @skip_on_fresh_install
+    def _reset_date(apps, schema_editor):
+        model = apps.get_model('sms', model_name)
+        min_date = model.objects.aggregate(min_date=Min("date_modified"))['min_date']
```

Comment:
nit: just curious if this could also be 
`min_date = model.objects.order_by('date_modified').first().date_modified`

I am thinking all records would have date_modified so no need to filter for where date_modified is not NULL.

Reply:
Yes, that would also work.